src/Sulu/Bundle/WebsiteBundle/DependencyInjection/SuluWebsiteExtension.php

```diff
@@ -149,5 +152,7 @@ public function load(array $configs, ContainerBuilder $container)
                 $loader->load('exception_controller.xml');
             }
         }
+
+        $this->configurePersistence($config['objects'], $container);
```

Comment:
would be nice to add an alias for the repository for autowiring:
https://github.com/sulu/sulu/blob/2.x/src/Sulu/Bundle/CategoryBundle/DependencyInjection/SuluCategoryExtension.php#L47-L54